src/sass/formapp.scss

```diff
@@ -3,12 +3,67 @@ body {
   font-family: 'ProximaSoft', 'Droid Sans', 'helvetica neue', 'arial', sans-serif !important;
   padding: 24px;
 }
+.col-form-label {
+  font-size: 18px;
+  font-weight: bold;
+}
 .form-control:disabled, .form-control[readonly] {
   background-color: inherit;
   border: none !important;
 }
-.btn:disabled {
-  opacity: .5;
+.block {
+  padding: 10px 20px;
+}
+@media screen {
+  .just-values th {
```

Comment:
are these being used in the forms?  Might be worth adding a comment to any custom classes that are not overriding formio or bootstrap.